src/PhpWord/TemplateProcessor.php

```diff
@@ -994,6 +982,12 @@ public function save()
         return $this->tempDocumentFilename;
     }
 
+    /** @return string */
+    public function getTempDocumentFilename()
```

Comment:
```suggestion
    public function getTempDocumentFilename(): string
```

Reply:
Done. I wasn't sure if the old versions of Php that you still support allowed scalar typehints; I'm glad to see that they do.